### PR TITLE
conferences app model

### PR DESCRIPTION
**Pull request by scoiatael**

### CODE REVIEW

File: conferences/models.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    )
+    end_date = models.DateField(
+        verbose_name=_('Last day')
+    )
+
+    active = models.BooleanField(
+        default=False
+    )
+
+    place_name = models.CharField(
```
Comment (scoiatael) [inline conferences/models.py]: TODO: extract to separate model

File: conferences/models.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        max_length=300
+    )
+    place_address = models.TextField()
+
+    banner = models.ImageField()
+
+    def __str__(self):
+        return 'In {} from {} to {}'.format(str(self.place_name),
+                                            self.start_date,
+                                            self.end_date)
```
Comment (scoiatael) [inline conferences/models.py:37]: TODO: add helper method to find most recent

File: conferences/models.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    start_date = models.DateField(
+        verbose_name=_('First day')
+    )
+    end_date = models.DateField(
+        verbose_name=_('Last day')
+    )
+
+    active = models.BooleanField(
+        default=False
+    )
```
Comment (scoiatael) [inline conferences/models.py:28]: TODO: Add validation that only one can be active at the time

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,2 @@
+test:
+	pep8 --exclude=env --max-line-length=120 .
```
Comment (scoiatael) [inline Makefile:2]: TODO: add unit test script

**Review by scoiatael — COMMENTED** [review]
Still in progress - but initial reviews are welcome.

File: conferences/models.py (hunk 1 above)
Comment (scoiatael) [reply conferences/models.py]: DONE.

File: conferences/models.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    banner = models.ImageField()
+
+    place = models.ForeignKey(Place)
+
+    def __str__(self):
+        return 'In {} from {} to {}'.format(str(self.place),
```
Comment (szewczykmira) [inline conferences/models.py:35]: Could look a little nicer as:

``` python
 return 'In {} from {} to {}'.format(
               str(self.place), self.start_date, self.end_date)
```

**Review by szewczykmira — APPROVED** [review]

**Pull request merged**